cppapi/client/devapi_base.cpp

```diff
@@ -1345,6 +1346,7 @@ DeviceData Connection::command_inout(string &command, DeviceData &data_in)
 // Now, try to execute the command
 //
 
+                //TODO extract class hierarchy
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this "TODO" comment. ([cpp:S1135](https://sonarcloud.io/organizations/tango-controls/rules#rule_key=cpp%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.tango-controls%3Acpp-tango%3Atango-9-lts&pullRequest=477&issues=AWWPm0Fu3Qu1svVwsabD&open=AWWPm0Fu3Qu1svVwsabD)

[ISSUE_KEY]: AWWPm0Fu3Qu1svVwsabD